studio/local_artifact_store.py

```diff
@@ -0,0 +1,64 @@
+import calendar
```

Comment:
This implementation is as straightforward as it gets:
basically, we just copy files around with no intent to optimize things.